pkg/utils/k8sutils/version.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved.
```

Comment:
any reason to create k8sutil under util?
We could have just called it version.go or k8s_version.go and directly put it under utils package.